lib/api/controller/auth.js

```diff
@@ -259,6 +260,44 @@ class AuthController extends NativeController {
       global.kuzzle.tokenManager.refresh(existingToken, token);
     }
 
+    // Only if the support of Browser Cookie as Jwt is enabled
+    // otherwise we should send a normal response because
+    // even if the SDK / Browser can handle the cookie,
+    // Kuzzle would not be capable of doing anything with it
+    if (global.kuzzle.config.security.supportBrowserCookieAsJwt) {
+      const isCookieOnly = request.input.args.cookieOnly;
+
+      if (isCookieOnly) {
```

Comment:
Redondant if, you should have both conditions in the same if. Also, since you use this at 3 different places you could DRY this code